### DISCUSSION TITLE
Make the REPL available in Pygame Zero mode

### DISCUSSION THREAD

**Discussion by ericclack** [mu-editor/mu#1490] [Ideas & New Features]
We're using Mu and Pygame Zero at Coder Dojo Brighton and it's a great environment for those new to Python :)

Is there any way to have the REPL available in Pygame Zero mode? We like to encourage our students to try out commands in the REPL to see how they work before they add them to their program. I know we can switch between modes to do this, but it would be nice to have the REPL button there at the top.

Any questions let me know, and happy to test too.

**Comment by ntoll** [comment]
The short answer is yes. I hope to have this feature in very soon (and thanks for the suggestion).
